src/sql/workbench/api/node/mainThreadModelViewDialog.ts

```diff
@@ -32,6 +39,15 @@ export class MainThreadModelViewDialog implements MainThreadModelViewDialogShape
 		throw new Error('Method not implemented.');
 	}
 
+	public $openEditor(modelViewId: string, title: string, position?: vscode.ViewColumn): Thenable<void> {
+		let input = new ModelViewInput(title, modelViewId);
+		let editorOptions = Object.assign({
```

Comment:
I think this is left over from my original code but you can remove `Object.assign` here and just leave the inner object since we're not exposing a way for the user to set the editor options (maybe we should in the future?)